msb/analysis/timestepinterpolator.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from msb.network.pubsub.types import Subscriber
+
+
+class PandasInterpolator:
```

Comment:
Consider renaming. Even though the interpolator used pandas internally, is this really what it is defined by?
E.g. `BatchInterpolator` or rather `BatchResampler`?